src/app/cli/src/coda.ml

```diff
@@ -486,7 +479,7 @@ let daemon logger =
        Web_pipe.run_service coda web_service ~conf_dir ~logger ;
        Coda_run.setup_local_server ?client_whitelist ?rest_server_port ~coda
          ~client_port () ;
-       Coda_run.run_snark_worker ~client_port run_snark_worker_action ;
+       Coda_run.run_snark_worker client_port ;
```

Comment:
`run_snark_worker_flag` is now unused. It should be inserted into a config somewhere I believe.

Reply:
If the snark_worker dies, I will kill the entire node. I will make sure that the snark_worker will be called lazy when there is `Some` public key assigned for the worker.

Reply:
Why specifically should we delay `run_snark_worker` until `replace_snark_work_key` is called? I don't see how it would help with loading the transaction snark proving keys.